advanced/Scripts/update.sh

```diff
@@ -62,7 +66,7 @@ GitCheckUpdateAvail() {
   fi
 
   # Change back to original directory
-  cd "${curdir}"
+  cd "${curdir}" || exit
```

Comment:
I'd fix that now, but that would reset all the current approvals, and it's really not breaking anything it's just extra steps that don't need to be taken.